src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx

```diff
@@ -159,6 +159,38 @@ describe('SelectMultiple', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('Value should be of same order as the order of selection', async () => {
```

Comment:
Can we swap the order of selections in this test so that we select something that would end up in a different order than alphabetical? For example Apple, Orange, then Banana